### PR TITLE
feat(renderer): indent_markers add a item icon

### PR DESCRIPTION
**Pull request by rammiah**
I add a item icon for indent_markers, make it possible to make nvim-tree view like tree command output.

When we config indent_markers to this:

```lua
render.indent_markers = {
    enable = true,
    icons = {
        corner = "└─",
        item = "├─",
        edge = "│ ",
        none = "  ",
    },
},
```

nvim-tree view:

image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -113,6 +113,7 @@ require'nvim-tree'.setup { -- BEGIN_DEFAULT_OPTS
       icons = {
         corner = "└ ",
         edge = "│ ",
+        item = "│ ",
```
Comment (kyazdani42) [inline README.md:116]: should be empty string by default :)

**Review by kyazdani42 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (rammiah) [reply README.md:116]: it is used to replace the innerest file's edge icon,  shouldn't be empty

Comment (kyazdani42) [reply README.md:116]: i see sorry

**Review by kyazdani42 — APPROVED** [review]

**Comment by alex-courtis** [comment]
nice

**Pull request merged**